checker/src/main/java/org/checkerframework/checker/lock/LockVisitor.java

```diff
@@ -329,6 +329,12 @@ protected boolean skipReceiverSubtypeCheck(
         return annotationSet;
     }
 
+    @Override
+    protected void checkConstructorResult(
+            AnnotatedExecutableType constructorType, ExecutableElement constructorElement) {
+        // Skip this check
```

Comment:
Why does this make sense for the Lock Checker? Can you make the comment more concrete?